packages/cyberstorm/src/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { IndexLink } from "./Links";
+
+type BreadcrumbsProps = PropsWithChildren<{
+  excludeHome?: boolean;
+}>;
+
+export const Breadcrumbs: React.FC<BreadcrumbsProps> = (props) => {
+  let nodes: ReactNode[] = !props.excludeHome ? [<DefaultHomeCrumb />] : [];
```

Comment:
Linter throwing this, on line 13 and 16. Though according to info here https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md , I'm not sure if this can be fixed and should just be disabled on these lines.
![image](https://user-images.githubusercontent.com/34424527/208832852-2259435e-2f77-42fe-94ae-734661489a01.png)

Reply:
AFAIK this probably does not have to be fixed, unless React does some weird stuff with them before even rendering. The key prop is used for React to map changes to existing elements in the DOM once they're mounted, and it's only required if there's more than one child in an element. If you look at how this component is rendered, you'll notice that the keys are set accordingly and this element never needs one since it gets wrapped by the Breadcrumb component.